tools/mo/openvino/tools/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -427,6 +427,8 @@ def add_net_rt_info(net: Element, meta_info: dict):
     else:
         meta = SubElement(net, 'rt_info')
         for key, value in meta_info.items():
+            if isinstance(value, dict) and value == {}:
+                continue
```

Comment:
Not clear why this change is needed? Please clarify

Reply:
This check is needed to prevent following line in IR:
```
<rt_info>
<conversion_parameters  />
</rt_info>
```
Such line leads to error of reading of rt_info. 
Before this fix `'input_model`' or `saved_model_dir `was always in conversion_parameters, so such line was impossible. But now '`input_model`' can be skipped and without this check it leads to empty section.

Reply:
Should we report such limitation to IR Frontend?
@ilyachur, fyi.

Reply:
It looks like incorrect format, at least for "conversion_parameters" case, so I don't think it is a bug. Not sure if such empty sections should be supported in general.

Reply:
okay, not principal concern.